client/components/domain-search-v2/free-domain-for-a-year-promo/style.scss

```diff
@@ -7,6 +7,7 @@
 }
 
 .free-domain-for-a-year-promo__image {
+	width: 140px;
```

Comment:
@bogiii I noticed that in the transition between empty and search results (or on load) the free domain promo image starts with no width and then shifts the content to the right. This is fixed here 👍🏻